rfcs/2024-03-28-MQTT_Default_Presets

```diff
@@ -0,0 +1,61 @@
+
+#Motivation
+[Why are we doing this? What problem does it solve or what use cases does it support? Discuss the benefits for the different components of the Meshtastic ecosystem, including end-users, developers, and core maintainers.]
+A single user can currently flood the local public network with out-of-region traffic that gets propagated through the mesh, cluttering up maps and reducing limited bandwidth on long/fast. By focusing on modem presets tailored to higher traffic (and shorter range) this new mesh can more easily handle the influx of traffic. The shorter range will be compensated for by the assumption that there will be more like-minded people in each region passing on the traffic to MQTT.
+
+#Ecosystem Impact
+[Explain how this change will affect the various components of the Meshtastic ecosystem, including firmware, phone, desktop, and web applications. Address any coordination required among these components.]
+The two separate "default" meshes in each region will have incompatible modem settings. 
+Private channel settings won’t be affected.
+No hardware or application changes other than setting the new modem preset which will enforce MQTT traffic off the default long/fast preset.
+
+
+#Protocol Buffer Changes
+[Detail the changes proposed to Meshtastic's protocol buffer definitions, if any. Explain how these changes will be managed and propagated across the different applications and firmware.]
+None.
+
+#Technical Details
+[Provide a more in-depth technical explanation of the proposed changes, focusing on the high-level architecture and how different components of the ecosystem will interact with these changes. This section should explain your proposed solution in enough detail that someone familiar with the Meshtastic ecosystem can understand the design and implementation of the feature.]
+None required. There will just be new modem presets when MQTT is turned on. Short/Fast or Medium/Fast would be good candidates.
+
+#Compatibility Considerations
+[Discuss how the proposed changes will affect backward compatibility across the ecosystem. Include strategies for handling compatibility issues. Discuss whether this change requires any version bumps.]
+All users wanting to continue to use MQTT will need to update to the latest firmware and the their local RF mesh will lose range (at the expense of increased bandwidth) so additional RF nodes may be required for the same coverage where internet is not available to fall back to.
```

Comment:
We can't enforce this for people running older firmware.

Reply:
@GUVWAF There is a non-elegant way we can, sort of. Will talk to you offline.